.docker/Dockerfile-build

```diff
@@ -18,30 +20,16 @@ COPY . .
 
 RUN go build -buildvcs=false -tags sqlite -o /usr/bin/keto .
 
-FROM alpine:3.18
-
-RUN addgroup -S ory; \
-    adduser -S ory -G ory -D  -h /home/ory -s /bin/nologin; \
-    chown -R ory:ory /home/ory
+#########################
 
-RUN apk --no-cache --latest upgrade &&\
-    apk --no-cache --upgrade --latest add ca-certificates
+FROM gcr.io/distroless/base-nossl-debian11:nonroot AS runner
```

Comment:
Does this come with CGO? We need it for SQLite

Reply:
Yes, the image comes with glibc -> https://github.com/GoogleContainerTools/distroless/blob/main/base/README.md :)